code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -212,6 +212,12 @@
 	. = ..()
 	if(methods & TOUCH)
 		exposed_mob.extinguish_mob() // extinguish removes all fire stacks
+	if(methods & VAPOR)
+		if(!isfelinid(exposed_mob))
+			return
+		exposed_mob.Stun(1) //cancels any do_after
```

Comment:
will this force them to drop what they're carrying?

will this affect felinids who are in hardsuits or the like?

Reply:
Swapped for a new status effect.